lib/middleware/assignActiveLicence.js

```diff
@@ -4,25 +4,28 @@ const logger = require('@financial-times/n-logger').default;
 const RE_FTCOM = /^FT\.com\s*/i;
 
 function _addLicenceAcquisitionData (licence) {
+	licence.displayName = null;
```

Comment:
why is this being null'd?

Reply:
Wanted to make sure the object has the same keys regardless of if anything is returned from the acquisition context. Also wanted to be explicit about a value in that scenario. I've changed to check for it's existence first so null is only set if an empty key is added.